src/fireflyClient.ts

```diff
@@ -826,8 +829,8 @@ export class FireflyClient {
    * @param symbol market symbol get information about
    * @returns user default leverage
    */
-  getUserDefaultLeverage = async (symbol: MarketSymbol) => {
-    const accData = await this.getUserAccountData();
+  getUserDefaultLeverage = async (symbol: MarketSymbol,parentAddress?:string) => {
```

Comment:
again if we make a field `parent` in class, we can simply call `this.getUserAccountData(this.parent)` no need to take as method argument. On class creation `this.parent = this.account.address` but user can update it using `setParentMethod` once done, all methods will work for the parent account.

Reply:
As discussed, sending undefined is okay.